src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonParameterInfoOfT.cs

```diff
@@ -85,10 +84,8 @@ public bool ReadJsonTyped(ref ReadStack state, ref Utf8JsonReader reader, out T
             bool success;
             bool isNullToken = reader.TokenType == JsonTokenType.Null;
 
-            if (isNullToken &&
-                ((!_converter.HandleNullValue && !state.IsContinuation) || Options.IgnoreNullValues))
+            if (isNullToken && !_converter.HandleNullValue && !state.IsContinuation)
```

Comment:
Can you remind me why we check for `IsContinuation`? If true, then we need to forward to the converter (because it may have run out of data on the previous read of the null value?)

Reply:
> because it may have run out of data on the previous read of the null value?

Yes.

I'll follow up with test coverage for this check in this method. The equivalent code in `JsonPropertyInfo<T>` (which reads and sets the property) is covered.